card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -37,7 +37,7 @@ def default_modal_content_args args
   end
 
   view :modal_body do |args|
-    _render_content(args)
+    _render_core(args)
```

Comment:
In that case, use the alias form (view :model_body, :core), it will do what you want with one fewer block defined.